clang/include/clang/AST/DeclCXX.h

```diff
@@ -598,7 +599,9 @@ class CXXRecordDecl : public RecordDecl {
     return !hasDefinition() || !isDynamicClass() || hasAnyDependentBases();
   }
 
-  void setIsParsingBaseSpecifiers() { data().IsParsingBaseSpecifiers = true; }
+  void setIsParsingBaseSpecifiers(bool to = true) {
```

Comment:
```suggestion
  void setIsParsingBaseSpecifiers(bool To = true) {
```

I am surprised clang-format didn't complain about starting with a lowercase for variables.
I can't think of a better name. I respond here if I do :).